pkg/nsx/services/inventory/builder.go

```diff
@@ -46,8 +46,10 @@ func (s *InventoryService) BuildPod(pod *corev1.Pod) (retry bool) {
 		if pod.Spec.NodeName != "" {
 			// retry when pod has Node but Node is missing in NodeInformer
 			retry = true
+			log.Error(err, "Cannot find node for Pod", "Pod", pod.Name, "Namespace", pod.Namespace, "Node", pod.Spec.NodeName, "retry", retry)
```

Comment:
Should Line 49/51 to be swapped?
Line 49 should be a Warn
Line 51 is a Error
Could you help to add return after Line 52?
Could you also help to update the Line40 with Warn?